core/store/schema.go

```diff
@@ -0,0 +1,17 @@
+package store
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func DumpSchema(dbURL string, args ...string) (string, error) {
+	args = append([]string{dbURL, "--schema-only"}, args...)
+	cmd := exec.Command("pg_dump", args...)
```

Comment:
Could just be a shell script under `tools/`?

Reply:
Yeah that would probably be better, especially if it ends up more complex - trying to rig up a sort to make it readable and definitely deterministic.